Mathmagician/Prime.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Mathmagician
+{
+    public class Prime
+    {
+        List<int> primes = new List<int>();
+        public Prime(int numOfPrimes)
+        {
+            ////////////////////////////primes ALL start with 2 (1 is not a prime number, only has 1 divisor(itself))
+            int lastPrime = 2;
+            Console.WriteLine(numOfPrimes);

```

Comment:
@VikingPaul Your Prime class shouldn't not be making a call to the `Console` class. Leave that to Program.cs